airbyte-integrations/connectors/source-looker/source_looker/streams.py

```diff
@@ -0,0 +1,532 @@
+        dt_func = f"date_time({dt.year}, {dt.month}, {dt.day}, {dt.hour}, {dt.minute}, {dt.second})"
+        client_id = self.get_query_client_id(stream_state)
+
+        # need to add the custom client_id value. It is used for filtering
+        # its value shouldn't be changed in the future
+        return {
+            "model": "i__looker",
+            "view": "history",
+            "limit": "10000",
+            "client_id": client_id,
+            "fields": [
+                "query.id",
+                "history.created_date",
+                "history.created_time",
+                "query.client_id",
+                "query.model",
+                "query.view",
+                "space.id",
+                "look.id",
+                "dashboard.id",
+                "user.id",
+                "history.query_run_count",
+                "history.total_runtime",
+            ],
+            "filters": {
+                "query.model": "-EMPTY",
+                "history.runtime": "NOT NULL",
+                "user.is_looker": "No",
+            },
+            "filter_expression": f"${{history.created_time}} > {dt_func}",
+            "sorts": [
+                "history.created_time",
+                "query.id",
+            ],
+        }
+
+    def parse_response(self, response: requests.Response, stream_slice: Mapping[str, Any], **kwargs: Any) -> Iterable[Mapping]:
+        records = response.json()
+        for i in range(len(records)):
+            record = records[i]
+            if record.get("looker_error"):
+                raise LookerException(f"Locker Error: {record['looker_error']}")
+            if (record.get("query.client_id") or "").startswith(self.airbyte_client_id_prefix):
+                # skip all native connector's requests
+                continue
+            # query.column_limit is used for filtering only
+            record.pop("query.client_id", None)
+
+            # convert date to ISO format: 2021-10-12 10:46 => 2021-10-12T10:46:00Z
+            record[self.cursor_field] = record.pop("history.created_time").replace(" ", "T") + "Z"
+
+            if i >= len(records) - 1:
+                self._is_finished = True
+            # convert history.created_date => history_created_date etc
+            yield {k.replace(".", "_"): v for k, v in record.items()}
+
+    def get_updated_state(self, current_stream_state: MutableMapping[str, Any], latest_record: Mapping[str, Any]) -> Mapping[str, Any]:
+        record_query_id = latest_record["query_id"]
+        if not self._is_finished and self._last_query_id == record_query_id:
+            if not self._last_query_id:
+                self._last_query_id = record_query_id
+            return current_stream_state
+        return {self.cursor_field: max((current_stream_state or {}).get(self.cursor_field) or "", latest_record[self.cursor_field])}
+
+
+class RunLooks(LookerStream):
+    """ "
```

Comment:
Nitpick. Extra quotation mark here.

```suggestion
    """
```